packages/neuron-wallet/src/services/node.ts

```diff
@@ -21,6 +21,15 @@ import { generateRPC } from '../utils/ckb-rpc'
 import startMonitor from './monitor'
 import { CKBLightRunner } from './light-runner'
 
+export enum VerifyCkbVersionResult {
+  Same,
+  Compatibility,
+  ShouldUpdate,
+  NotCompatibility,
+}
+
+const supportCkbVersions = ['0.110', '0.109']
```

Comment:
How about defining it in an independent file so the compatibility table can be shared elsewhere.

Reply:
Especially to be synced to the troubleshooting page

Reply:
The troubleshooting page will show all Neuron versions, but here it will only show the current Neuron version.
And how to share it to the troubleshooting page? They are different projects.

Reply:
It won't be considerable to include a complete compatibility table but it helps users to check compatibility without visiting another project.

The troubleshooting page fetches contents, e.g. issues and discussions from this project, so it can fetch the compatibility table similarly, to render the compatibility table on page.

---

CSV is preferable to be used for the compatibility table, e.g. https://raw.githubusercontent.com/citizenlab/test-lists/master/lists/00-LEGEND-category_codes.csv

Reply:
|  Neuron  | CKB |
|  ----  | ----  |
| 0.110  | 0.109;0.110; |
| 0.106  | 0.105;0.106;0.107;0.108; |
| 0.103 | 0.103;0.104;|

How about defining it like this?

Reply:
> Neuron	CKB
> 0.110	0.109;0.110;
> 0.106	0.105;0.106;0.107;0.108;
> 0.103	0.103;0.104;
> How about defining it like this?

It's hard to read and parse, I would suggest https://gist.github.com/Keith-CY/f76ebd3ae0f89e5a5f6dc85e29096144